contacts.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="ru-RU">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Контакты</title>
+    <link rel="stylesheet" href="css/style.css">
+	<style>
+			body{
+				 background: url(images/fon3.jpg )  no-repeat;
+				 background-position: top center;
+				 background-attachment: fixed;
+				 background-size: cover ;
+				}
+		  </style>
+</head>
+<body>
+        <header>
+                <h1 align="center " class="selection-title">Heorhi Buklaha</h1>
+                <address>
+                    <div>City : Minsk</div>
+                    <div>Pone : +37533644-79-89</div>
+                    <div>Email : buklaga.98@yandex.ru</div>
+                </address>
+                <nav>
+                        <ul id="nav">
```

Comment:
Code style is broken: the number of indents for nested elements. See here and below.
Read more in http://codeguide.academy/html-css.html